setup.py

```diff
@@ -47,7 +47,8 @@
 ]
 
 install_requires = [
-    'mosaicml[nlp,streaming,wandb]>=0.14.1,<0.15',
+    'composer[nlp,streaming,wandb]>=0.14.1,<0.15',
```

Comment:
can we remove the `streaming` part from here? Would prefer to just have one line installing streaming